app/scripts/controllers/user-storage/encryption.ts

```diff
@@ -0,0 +1,117 @@
+import sjcl from 'sjcl';
+
+// Encryption / Decryption taken from - https://github.com/luke-park/SecureCompatibleEncryptionExamples/blob/master/TypeScript/SCEE-sjcl.ts
+class EncryptorDecryptor {
+  #BITS_PER_WORD = 32;
+
+  #ALGORITHM_NONCE_SIZE = 3; // 32-bit words.
+
+  #ALGORITHM_KEY_SIZE = 4; // 32-bit words.
+
+  #PBKDF2_SALT_SIZE = 4; // 32-bit words.
+
+  #PBKDF2_ITERATIONS = 32767;
```

Comment:
This number seems rather low. How was it chosen? On mobile, [we are bumping](https://github.com/MetaMask/metamask-mobile/blob/4a4fbe6fdbb4bbfc30f5aa58b49adf96167d0b13/app/core/Encryptor/constants.ts#L12) the default to 900,000 iterations.

Reply:
Directly taken from the snippet above:
https://github.com/luke-park/SecureCompatibleEncryptionExamples/blob/master/TypeScript/SCEE-sjcl.ts

We can bump the number of iterations!

Reply:
Let's not spend time on this, `sjcl` should not be used.

Reply:
I've tested replacing this with either `@noble/*` or with `@metamask/browser-passworder`. Most likely we will adopt `@noble/*` will update this soon.

Reply:
This has been updated to use `@noble/*`. We opted for this as `@metamask/browser-passworder` is browser focused (will not work on Mobile unless polyfilled).